doc/release-notes.rst

```diff
@@ -610,6 +610,130 @@ Upgrading from pre-Jewel releases (like Hammer)
+  foo').
+
+* The 'rgw rest getusage op compat' config option allows you to dump
+  (or not dump) the description of user stats in the S3 GetUsage
+  API. This option defaults to false.  If the value is true, the
+  reponse data for GetUsage looks like::
+
+    "stats": {
+                "TotalBytes": 516,
+                "TotalBytesRounded": 1024,
+                "TotalEntries": 1
+             }
+
+  If the value is false, the reponse for GetUsage looks as it did before::
+
+    {
+         516,
+         1024,
+         1
+    }
+
+* The 'osd out ...' and 'osd in ...' commands now preserve the OSD
+  weight.  That is, after marking an OSD out and then in, the weight
+  will be the same as before (instead of being reset to 1.0).
+  Previously the mons would only preserve the weight if the mon
+  automatically marked and OSD out and then in, but not when an admin
+  did so explicitly.
+
+* The 'ceph osd perf' command will display 'commit_latency(ms)' and
+  'apply_latency(ms)'. Previously, the names of these two columns are
+  'fs_commit_latency(ms)' and 'fs_apply_latency(ms)'. We remove the
+  prefix 'fs\_', because they are not filestore specific.
+
+* Monitors will no longer allow pools to be removed by default.  The
+  setting mon_allow_pool_delete has to be set to true (defaults to
+  false) before they allow pools to be removed.  This is a additional
+  safeguard against pools being removed by accident.
+
+* If you have manually specified the monitor user rocksdb via the
+  ``mon keyvaluedb = rocksdb`` option, you will need to manually add a
+  file to the mon data directory to preserve this option::
+
+     echo rocksdb > /var/lib/ceph/mon/ceph-`hostname`/kv_backend
+
+  New monitors will now use rocksdb by default, but if that file is
+  not present, existing monitors will use leveldb.  The ``mon
+  keyvaluedb`` option now only affects the backend chosen when a
+  monitor is created.
+
+* The 'osd crush initial weight' option allows you to specify a CRUSH
+  weight for a newly added OSD.  Previously a value of 0 (the default)
+  meant that we should use the size of the OSD's store to weight the
+  new OSD.  Now, a value of 0 means it should have a weight of 0, and
+  a negative value (the new default) means we should automatically
+  weight the OSD based on its size.  If your configuration file
+  explicitly specifies a value of 0 for this option you will need to
+  change it to a negative value (e.g., -1) to preserve the current
+  behavior.
+
+* The static libraries are no longer included by the debian
+  development packages (lib*-dev) as it is not required per debian
+  packaging policy.  The shared (.so) versions are packaged as before.
+
+* The libtool pseudo-libraries (.la files) are no longer included by
+  the debian development packages (lib*-dev) as they are not required
+  per https://wiki.debian.org/ReleaseGoals/LAFileRemoval and
+  https://www.debian.org/doc/manuals/maint-guide/advanced.en.html.
```

Comment:
The "libtool pseudo-libraries are no longer included by the debian . . ." RN can be dropped.